data/lang/module-scout/en.json

```diff
@@ -0,0 +1,482 @@
+    "description" : "",
+    "message" : "DISTANCE REACHED. MAPPING. Remain below altitude ceiling until scan completed."
+  },
+  "EXCELLENT_I_AWAIT_YOUR_REPORT" : {
+    "description" : "",
+    "message" : "Excellent. I will await your report."
+  },
+  "HOW_DOES_IT_WORK" : {
+    "description" : "",
+    "message" : "How does it work."
+  },
+  "INTROTEXT_COMPLETED_1" : {
+    "description" : "",
+    "message" : "Hello! \nIf you are seeing this is because the data were obtained and confirmed what we suspected.\nSee you at:"
+  },
+  "INTROTEXT_COMPLETED_2" : {
+    "description" : "",
+    "message" : "Hello! \nIf you are seeing this is because the data were successfully mapped. Will be very useful to my friends.\n To complete the mission, you must go to:"
+  },
+  "INTROTEXT_COMPLETED_3" : {
+    "description" : "",
+    "message" : "Greetings again. Apparently, the mission has been a success.\nYou must carry this information to:"
+  },
+  "INTROTEXT_COMPLETED_4" : {
+    "description" : "",
+    "message" : "Hello! \nWhat you are seeing this means that the data collected are very interesting and will be very useful to my editor.\n The delivery shall be in:"
+  },
+  "INTROTEXT_COMPLETED_5" : {
+    "description" : "",
+    "message" : "Hello! \nIf you are seeing this is that the mission is successful now. The data will be very useful.\n Your reward awaits you in:"
+  },
+  "INTROTEXT_COMPLETED_6" : {
+    "description" : "",
+    "message" : "You are viewing this because, the first part of mission has been successful.\n To complete it should be directed to:"
+  },
+  "INTROTEXT_COMPLETED_7" : {
+    "description" : "",
+    "message" : "If you are seeing this, the first part of mission seems to have been successful. To complete it you are directed to go to:"
+  },
+  "INTROTEXT_COMPLETED_8" : {
+    "description" : "",
+    "message" : "xxx"
+  },
+  "INTROTEXT_COMPLETED_9" : {
+    "description" : "",
+    "message" : "xxx"
+  },
+  "PLEASE_HAVE_THE_DATA_BACK_BEFORE" : {
+    "description" : "",
+    "message" : "Please have the data back before {date}"
+  },
+  "MAPPING" : {
+    "description" : "As in cartography",
+    "message" : "Mapping"
+  },
+  "MAPPING_INTERRUPTED_ORBITAL" : {
+    "description" : "",
+    "message" : "MAPPING INTERRUPTED. Too far from target. Orbit correction needed."
+  },
+  "MAPPING_INTERRUPTED_SURFACE" : {
+    "description" : "",
+    "message" : "MAPPING INTERRUPTED. Too far from target. Flight ceiling correction needed."
+  },
+  "MAPPING_PAUSED" : {
+    "description" : "",
+    "message" : "MAPPING SUSPENDED. Data incomplete. Further mapping required."
+  },
+  "OBJECTIVE" : {
+    "description" : "",
+    "message" : "Objective:"
+  },
+  "ORBITAL_SCAN" : {
+    "description" : "",
+    "message" : "Orbital Scan"
+  },
+  "SURFACE_SCAN" : {
+    "description" : "",
+    "message" : "Surface Scan"
+  },
+  "AN_ORBITAL_SCAN" : {
+    "description" : "",
+    "message" : "an Orbital scan"
```

Comment:
I think this should be lower case
"an orbital scan"

Reply:
I'm thinking the whole sentence into which that variable fits should be a string, will give translators more flexibility in other languages.  `"You need to perform {type} of {body}.`

Reply:
I agree.

Reply:
since you're going through the strings, you can gather the changes in a PR. But try to minimize changes, each change triggers request for re-translation in all languages, so only fix things that are very wrong, rather than "I'd prefer to word it thusly"

Reply:
This was left upper-case because Orbital / Surface are intended as "proper" nouns, as they reference specific game concepts (orbital / surface scanner) and are intended to stand out from the surrounding sentence (for lack of both a way to link words to game tutorials and also a database of game information accessible through the UI). I leave it to your discretion as to whether that should remain or not.